### PR TITLE
feat(controls): Blockquote, migrate knobs to controls - FRONT-1848

### PR DESCRIPTION
**Pull request by papegaill**

### CODE REVIEW

File: src/implementations/twig/components/blockquote/blockquote.story.js [hunk 1]
```diff
@@ -1,28 +1,61 @@
...
-const prepareQuote = (data) => {
-  data.citation = text('citation', data.citation, tabLabels.required);
-
-  data.author = text('author', data.author, tabLabels.required);
+const getArgTypes = () => {
+  const argTypes = {};
+  argTypes.author = {
+    name: 'Author',
+    type: { name: 'string', required: true },
+    description: 'Author of the citation',
```
Comment (emeryro) [inline src/implementations/twig/components/blockquote/blockquote.story.js]: for some reasons, I don't see the description on storybook
image
Is it the same for you?

File: src/implementations/twig/components/blockquote/blockquote.story.js [hunk 2]
```diff
@@ -1,28 +1,61 @@
...
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
+      type: 'text',
+    },
+  };
+  argTypes.citation = {
```
Comment (emeryro) [inline src/implementations/twig/components/blockquote/blockquote.story.js]: a small detail, but can we put the citation before the author in the control list?
That would reflect the normal reading order of the component

**Review by emeryro — CHANGES_REQUESTED** [review]
please fix the conflict reported (I'm not sure what has to be kept here)

File: src/implementations/twig/components/blockquote/blockquote.story.js (hunk 1 above)
Comment (papegaill) [reply src/implementations/twig/components/blockquote/blockquote.story.js]: Thks forgot to return `argTypes` in getArgTypes function 👍

**Review by emeryro — APPROVED** [review]

**Pull request merged**